### DISCUSSION TITLE
Filenames normalisation

### DISCUSSION THREAD

**Discussion by artola** [webpack/webpack#14141] [Q&A]
I have upgraded from v4 to v5 and found a problem with the code using `babel-plugin-relay`, affecting only Windows.
I have created a repo an I can see that it happens with or without TS (ts-loader), but might be similar to https://blog.johnnyreilly.com/2021/04/20/ts-loader-goes-webpack-5/
I guess that might be solved re-writting in the loader but should not be done in the babel-plugin else it should work differently in each environment.

@sokra I would appreciate your feedback, about how/where to proceed on this issue: [relay]( https://github.com/facebook/relay/issues/3272#issuecomment-911654887)

**Comment by alexander-akait** [comment]
Sorry, we can't control external code and I don't think you have the same problem

**Reply by alexander-akait** [reply]
Can you provide example of the problem?

**Reply by artola** [reply]
Thanks. Here a repo, the same code works on posix but fails on windows.

https://github.com/artola/bug-windows.git

```sh
yarn
yarn build
```

![image](https://user-images.githubusercontent.com/11500763/132049149-9826e349-b30c-499b-81fa-5483e7d14620.png)

**Comment by artola** [comment]
Here the explanation of @sokra  https://github.com/facebook/relay/issues/3272#issuecomment-912781359

**Reply by alexander-akait** [reply]
As you can see you have `./..\`, nothing to normalize here, you can't use `\` in URL due spec

**Reply by alexander-akait** [reply]
And yes, we say the same here https://github.com/facebook/relay/issues/3272#issuecomment-912781359
